### ISSUE TITLE
[Bug Report] Illegal Invocation in JsQuery

### ISSUE THREAD

**Issue by qbx2** [openblocks-dev/openblocks#266]
When I run a jsQuery with script=`return btoa('1')`, it fails with the error message `Illegal invocation`. My colleague said it worked in v1.1.4 by the way. Anyway, I found a workaround which is to use script=`return btoa.call(null, '1')` to set `this` parameter to `null`, instead of `undefined`.

**Comment by qbx2** [comment]
I think this happens only in dockerized environment.

**Comment by lmx1117** [comment]
> When I run a jsQuery with script=`return btoa('1')`, it fails with the error message `Illegal invocation`. My colleague said it worked in v1.1.4 by the way. Anyway, I found a workaround which is to use script=`return btoa.call(null, '1')` to set `this` parameter to `null`, instead of `undefined`.

are you using 1.1.5 self-hosted OB?

**Comment by qbx2** [comment]
@lmx1117 I use self-hosted, but the issue also occurs in cloud, too.

**Comment by qbx2** [comment]
I’m sorry, it only occurs in self-hosted.

**Comment by lmx1117** [comment]
> I’m sorry, it only occurs in self-hosted.

Got it, thank you, we will check it asap

**Comment by lmx1117** [comment]
@qbx2 Hi please update to the [beta](https://hub.docker.com/layers/openblocksdev/openblocks-ce/beta/images/sha256-b3f68e27305e8993e509a79e06fa17c4abece6386ccff87accf0c5ff6870c612?context=explore) to see if this is fixed, looking forward to your feedback

**Comment by qbx2** [comment]
No, it isn't fixed. Same error message.

**Comment by qbx2** [comment]
Tested with:
```
REPOSITORY                    TAG       IMAGE ID       CREATED      SIZE
openblocksdev/openblocks-ce   beta      e7d6a721e5b7   2 days ago   1.38GB
```

**Comment by qbx2** [comment]
I'll try with explicit sha256 digest: docker run -it --platform linux/amd64 -p 3000:3000 openblocksdev/openblocks-ce@sha256:b3f68e27305e8993e509a79e06fa17c4abece6386ccff87accf0c5ff6870c612

**Comment by qbx2** [comment]
Not working anyway :/

**Comment by lmx1117** [comment]
@qbx2 sorry about that 🥲, we will check it again

**Comment by YoJayz** [comment]
Hi @qbx2, we've released Openblocks [v1.1.6](https://github.com/openblocks-dev/openblocks/releases/tag/v1.1.6) and this issue has been fixed. 
Please check again and thanks for your patience.

**Comment by qbx2** [comment]
@YoJayz I just tried v1.1.6 and confirmed that it works. Thank you for support!
